### DISCUSSION TITLE
Has anybody needed to use the --no-half?

### DISCUSSION THREAD

**Discussion by Setero259** [lllyasviel/stable-diffusion-webui-forge#134] [Q&A]
I tried to gen a normal image and got the black output but without an error. 
But when i put the --no-half on command ARG it worked fine.  
Isn't --no-half already implemented?
Any explanation?

**Comment by 49RpK5dY** [comment]
You have to use --all-in-fp32 instead if you used --no-half in a1111.

**Reply by TigonTX** [reply]
OMG, thank you! I've been fighting this for a week. Reinstalled 5 or 6 times

**Reply by bdjimmy** [reply]
--no-half-vae

**Reply by bdjimmy** [reply]
--no-half-controlnet
